src/calliope_pathways/models/italy_stationary/pre_processing/parse_lombardi.py

```diff
@@ -8,7 +8,7 @@
 from scipy.special import gamma
 from yaml import safe_load
 
-# >> Model setup >> User configurable
+# -> Model setup -> User configurable
```

Comment:
Add a TODO (or make the change) of moving these into a YAML config that someone could use instead of having to dive into the code? Similar to your comment in PPM

Reply:
Added the TODO, but I am not sure if the effort is worth it in this case... it's too specific.

For ppm it does because it _could_ become a generic thing we call for all Euro-Calliope stuff (if I find a good way of filling the missing hydro storage capacity).